site/content/docs/5.2/components/pagination.md

```diff
@@ -141,6 +141,26 @@ To avoid pagination wrapping or overflowing, we limit the number or pagination i
   </ul>
 </nav>
 {{< /example >}}
+
+## Dark variant
+
+Pagination comes with a dark variant: `.pagination-dark`.
+
+{{< example class="bg-dark" >}}
+<nav aria-label="Dark page navigation example">
+  <ul class="pagination pagination-dark">
+    <li class="page-item disabled">
+      <a class="page-link">Previous</a>
+    </li>
+    <li class="page-item"><a class="page-link" href="#">1</a></li>
+    <li class="page-item"><a class="page-link" href="#">2</a></li>
+    <li class="page-item"><a class="page-link" href="#">3</a></li>
```

Comment:
It could be interesting to add an `active` class on one of the 3 numbered items to illustrate the look of a selected item.

Reply:
You're totally right, fixed via last commit !